storage/watchable_store_test.go

```diff
@@ -222,3 +239,118 @@ func TestUnsafeAddWatching(t *testing.T) {
 		}
 	}
 }
+
+func TestNewMapWatchingToEventMap(t *testing.T) {
+	k0, k1, k2 := []byte("foo0"), []byte("foo1"), []byte("foo2")
+	v0, v1, v2 := []byte("bar0"), []byte("bar1"), []byte("bar2")
+
+	testWatchings := []*watching{
+		{key: k0},
+		{key: k1},
+		{key: k2},
+	}
+
+	testEvents := []storagepb.Event{
```

Comment:
Will change that as well. Thanks

On Thu, Dec 31, 2015 at 11:28 PM, Xiang Li <notifications@github.com> wrote:

> In storage/watchable_store_test.go
> <https://github.com/coreos/etcd/pull/4091#discussion_r48676601>:
>
> > @@ -222,3 +239,118 @@ func TestUnsafeAddWatching(t *testing.T) {
> >  		}
> >  	}
> >  }
> > +
> > +func TestNewMapWatchingToEventMap(t *testing.T) {
> > +	k0, k1, k2 := []byte("foo0"), []byte("foo1"), []byte("foo2")
> > +	v0, v1, v2 := []byte("bar0"), []byte("bar1"), []byte("bar2")
> > +
> > +	testWatchings := []*watching{
> > +		{key: k0},
> > +		{key: k1},
> > +		{key: k2},
> > +	}
> > +
> > +	testEvents := []storagepb.Event{
>
> just evs?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/etcd/pull/4091/files#r48676601>.
>



-- 
Sincerely,
Gyu-Ho Lee